src/test/java/APICalls/PostCall.java

```diff
@@ -0,0 +1,38 @@
+package APICalls;
+
+import JacksonModels.Response;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+
+
+import java.io.*;
+import java.util.Properties;
+
+import static utility.PropertyManager.getProperties;
+
+public class PostCall extends BaseCall {
+
+    protected static String POST_URL = "https://api.trello.com/1/boards/?name=test_board&key="+ apiKey+"&token="+ apiToken;
+    protected final CloseableHttpResponse httpResponse =  callPost();
+    protected final Response post = (Response) deserializeObject(httpResponse);
+
+    public PostCall() throws IOException {
+    }
+
+    protected static CloseableHttpResponse callPost() throws IOException {
+        try (CloseableHttpClient httpClient = HttpClients.createDefault();) {
+            HttpPost httpPost = new HttpPost(POST_URL);
+            return  httpClient.execute(httpPost);
+        }
+    }
+    protected void setNewID() throws IOException {
```

Comment:
setNewID method writes to a properties file, which seems out of scope for an API call class
it’s better to handle such operations in a separate utility class or in the PropertyManager